### ISSUE TITLE
Yarn: add to PATH

### ISSUE THREAD

**Issue by pascalgrimaud** [jhipster/jhipster-devbox#76]
As mentionned here https://github.com/jhipster/generator-jhipster/issues/5343#issuecomment-283897684

the var $PATH need to be updated with
```
echo 'export PATH="$PATH:`yarn global bin`:$HOME/.config/yarn/global/node_modules/.bin"' >> /home/vagrant/.zshrc
```
 (it's not to forget to do it)

**Comment by PierreBesson** [comment]
Also we need to install Yarn with proper install and not NPM.

**Comment by pascalgrimaud** [comment]
yes, but we need to check if it's ok to install with root user?
Or maybe we have to switch to user vagrant before launching the install

**Comment by pascalgrimaud** [comment]
@PierreBesson : look my other PR, I kept the install Yarn with NPM. I didn't manage to use ` curl -o- -L https://yarnpkg.com/install.sh | bash` as it seems to install only for the current user (so for the root user, instead of Vagrant). So let keep like this for the moment.
Don't hesitate if you find a better way!

**Comment by jdubois** [comment]
I also tried the installation with the .sh script, as it's the recommended
install procedure, but I'm not convinced:

- I don't understand why this would be more stable, as the curl request can
fail as well as the npm request (so the Yarn installation procedure
instructions look bogus to me)
- We already rely on NPM, so I wouldn't use yet another method to download
stuff - that's more systems that need to be reachable during the install,
so more risk of trouble.

Bottom line, let's keep the npm install for Yarn.

Le 4 mars 2017 3:02 AM, "Pascal Grimaud" <notifications@github.com> a
écrit :
